test/runtests.jl

```diff
@@ -43,6 +46,17 @@ end
         @test hssrank(hssC) == 3
     end;
 
+    @testset "random access" begin
+        I = rand(1:size(A,1), 10)
+        J = rand(1:size(A,1), 10)
+        function testAccuracy(expected, result)
```

Comment:
just for readability and style, could you make this an inline function declaration? So
`test_accuracy(target, result) = ....`. Feel free to break the line if it's too long.

Reply:
Hi,
 It is done in the last commit, 
Best, 
Baptiste